src/settings/toml/builder.rs

```diff
@@ -14,8 +15,6 @@ const UPLOAD_DIR: &str = "dist";
 #[serde(deny_unknown_fields)]
 pub struct Builder {
     pub command: Option<String>,
-    #[serde(default = "project_root")]
-    pub cwd: PathBuf,
```

Comment:
Why do we need to remove `cwd`? Even if not super useful, this config option made it into a stable release and we'll break folks who started using it if we remove it.